### PR TITLE
Added recipe strawberry-daiquiri

### PR DESCRIPTION
**Pull request by Kash2000**
The recipe was added using a JSON file named: strawberry-daiquiri
The image was added using JPG file named: strawberry-daiquiri

### CODE REVIEW

File: strawberry-daiquiri.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+    "name": "<Strawberry Daiquiri>",
```
Comment (alfg) [inline strawberry-daiquiri.json]:
```suggestion
    "name": "Strawberry Daiquiri",
```

Please remove all the `<`  and `>` in this file as they are just token placeholders.

File: strawberry-daiquiri.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        "<Pour in lime juice, lemon juice, rum and lemon-lime soda.>",
+        "<Blend until smooth.>"
+        "<Pour into glasses and serve.>"
+    ],
+    "image": "<strawberry-daiquiri.jpg>",
+    "source": "<https://www.allrecipes.com/recipe/32352/best-strawberry-daiquiri/>",
+    "keywords": [
+        "<keywords if available>",
+        "...",
+    ]
```
Comment (alfg) [inline strawberry-daiquiri.json]: Missing keywords? Add a few if possible.

File: strawberry-daiquiri.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    ],
+    "directions": [
+        "<In a blender, combine ice, sugar and strawberries.>",
+        "<Pour in lime juice, lemon juice, rum and lemon-lime soda.>",
+        "<Blend until smooth.>"
+        "<Pour into glasses and serve.>"
+    ],
+    "image": "<strawberry-daiquiri.jpg>",
+    "source": "<https://www.allrecipes.com/recipe/32352/best-strawberry-daiquiri/>",
+    "keywords": [
```
Comment (Kash2000) [inline strawberry-daiquiri.json]: I am not sure what to put for keywords

File: strawberry-daiquiri.json (hunk 1 above)
Comment (Kash2000) [reply strawberry-daiquiri.json]: I have made all the changes. Added the picture and the keywords. Sorry for the previous comments, I am very new to this. Also, how would I know if my pull request gets accepted?

**Comment by alfg** [comment]
One more change to move the JSON file to `src/recipes/`.

File: strawberry-daiquiri.json (hunk 3 above)
Comment (alfg) [reply strawberry-daiquiri.json]: strawberry and daiquiri would be fine.

File: strawberry-daiquiri.json (hunk 1 above)
Comment (alfg) [reply strawberry-daiquiri.json]: You'll know when it gets accepted when it gets merged.

**Pull request closed without merging**